lib/node_modules/@stdlib/array/complex64/lib/main.js

```diff
@@ -2499,6 +2499,86 @@ setReadOnly( Complex64Array.prototype, 'toString', function toString() {
 	return out.join( ',' );
 });
 
+/**
+* Returns a new array iterator object that iterates the value of each item in a typed array.
+*
+* @name values
+* @memberof Complex64Array.prototype
+* @type {Function}
+* @throws {TypeError} `this` must be a complex number array
+* @returns {Iterator} iterator
+*
+* @example
+* var realf = require( '@stdlib/complex/realf' );
+* var imagf = require( '@stdlib/complex/imagf' );
+* var arr = new Complex64Array( 2 );
+*
+* arr.set( [ 1.0, -1.0 ], 0 );
+* arr.set( [ 2.0, -2.0 ], 1 );
+*
+* var iter = arr.values();
+*
+* var v = iter.next().value;
+* // returns <Complex64>
+*
+* var re = realf( v );
+* // returns 1.0
+*
+* var im = imagf( v );
+* // returns -1.0
+*
+* v = iter.next().value;
+* // returns <Complex64>
+*
+* re = realf( v );
+* // returns 2.0
+*
+* im = imagf( v );
+* // returns -2.0
+*
+* var bool = iter.next().done;
+* // returns true
+*/
+setReadOnly( Complex64Array.prototype, 'values', function values() {
+	var iter;
+	var FLG;
+	var buf;
+	var len;
+	var i;
+
+	if ( !isComplexArray( this ) ) {
+		throw new TypeError( 'invalid invocation. `this` is not a complex number array.' );
+	}
+	buf = this._buffer;
+	len = this._length;
+	i = -1;
+	FLG = true;
+	iter = {};
+	setReadOnly( iter, 'next', next );
+	return iter;
+
+	/**
+	* Returns an iterator protocol-compliant object.
+	*
+	* @private
+	* @returns {Object} iterator protocol-compliant object
+	*/
+	function next() {
+		i += 1;
+		if ( FLG && i < len ) {
```

Comment:
What is the purpose of `FLG` here? It seems the exit strategy is satisfied with `i < len`.

Reply:
@Jaysukh-409 This comment has still not been addressed. The conditional should be `( FLG || i >= len )`. See https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/ndarray/iter/values/lib/main.js.

Reply:
This logic is still incorrect, as `FLG` should not be initially `true`.